cache/modules/wc_org_cloudfront/distribution.tf

```diff
@@ -0,0 +1,188 @@
+
+    allowed_methods        = local.all_methods
+    cached_methods         = local.stateless_methods
+    viewer_protocol_policy = "redirect-to-https"
+
+    cache_policy_id          = var.cache_policies["weco-apps"]
+    origin_request_policy_id = var.request_policies["host-query-and-toggles"]
+
+    dynamic "lambda_function_association" {
+      for_each = local.lambda_associations
+      content {
+        event_type = lambda_function_association.value.event_type
+        lambda_arn = lambda_function_association.value.lambda_arn
+      }
+    }
+  }
+
+  # Works
+  ordered_cache_behavior {
+    path_pattern     = "/works*"
+    target_origin_id = local.alb_origin_id
+
+    allowed_methods        = local.stateless_methods
+    cached_methods         = local.stateless_methods
+    viewer_protocol_policy = "redirect-to-https"
+
+    cache_policy_id          = var.cache_policies["weco-apps"]
+    origin_request_policy_id = var.request_policies["host-query-and-toggles"]
+
+    dynamic "lambda_function_association" {
+      for_each = local.lambda_associations
+      content {
+        event_type = lambda_function_association.value.event_type
+        lambda_arn = lambda_function_association.value.lambda_arn
+      }
+    }
+  }
+
+  # Account
+  ordered_cache_behavior {
+    path_pattern     = "/account*"
+    target_origin_id = local.alb_origin_id
+
+    allowed_methods        = local.all_methods
+    cached_methods         = local.stateless_methods
+    viewer_protocol_policy = "redirect-to-https"
+
+    cache_policy_id          = var.cache_policies["Managed-CachingDisabled"]
+    origin_request_policy_id = var.request_policies["Managed-AllViewer"]
+  }
+
+  # Images
+  ordered_cache_behavior {
+    path_pattern     = "/images*"
+    target_origin_id = local.alb_origin_id
+
+    allowed_methods        = local.stateless_methods
+    cached_methods         = local.stateless_methods
+    viewer_protocol_policy = "redirect-to-https"
+
+    cache_policy_id          = var.cache_policies["weco-apps"]
+    origin_request_policy_id = var.request_policies["host-query-and-toggles"]
+
+    dynamic "lambda_function_association" {
+      for_each = local.lambda_associations
+      content {
+        event_type = lambda_function_association.value.event_type
+        lambda_arn = lambda_function_association.value.lambda_arn
+      }
+    }
+  }
+
+  # This is for the data fetching routes used in NextJs's getServerSideProps
+  # see: https://nextjs.org/docs/basic-features/data-fetching#getserversideprops-server-side-rendering
+  ordered_cache_behavior {
+    path_pattern     = "/_next/data/*"
+    target_origin_id = local.alb_origin_id
+
+    allowed_methods        = local.stateless_methods
+    cached_methods         = local.stateless_methods
+    viewer_protocol_policy = "redirect-to-https"
+
+    cache_policy_id          = var.cache_policies["toggle-cookies-only"]
+    origin_request_policy_id = var.request_policies["host-query-and-toggles"]
```

Comment:
Is this unnecessarily fragmenting our _next cache by toggles? AIUI, everyone gets the same bundle but the toggle causes different behaviour. Or have I understood that wrong?

Reply:
I think you're right, but I'll probably leave this as-is because I don't want to fragment policies too much and the vast majority of public users will have the same toggles (ie none)